config/settings/development.yml

```diff
@@ -1,3 +1,5 @@
 ---
 :log:
   :level_rails: debug
+:development:
+  :use_vim_broker_in_console: false
```

Comment:
This setting must also exist in the config/settings.yml with whatever default we would choose in prod (and test I guess), so probably true.  We need this because we are trying to avoid "hidden settings", which this would be without the change to the base settings.yml.

Reply:
Thinking about it further...since this setting is vmware specific, I think this should be in `manageiq-providers-vmware/config/settings.yml` and `manageiq-providers-vmware/config/settings/development.yml`.  Those get sucked in as templates, so it will satisfy the "hidden settings" requirement while also keeping it "pluggable"

Reply:
Please move this to the top, alphabetically.